frame/staking/src/lib.rs

```diff
@@ -829,6 +929,22 @@ decl_error! {
 		NoMoreChunks,
 		/// Can not rebond without unlocking chunks.
 		NoUnlockChunk,
+		/// The submitted result is received out of the open window.
```

Comment:
I am not really happy about adding this, but AFAIK there is no way to nest enums here right? Ideally I would just add `PhragmenErr(inner)` here and the rest somewhere else. Alternatives? @bkchr  @shawntabrizi ?

Reply:
Yes there is currently not better alternative.